src/controllers/itemdetailpage.js

```diff
@@ -418,13 +426,17 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "userSetti
         var offset = parentNameLast ? ".25em" : ".5em";
 
         if (html && !parentNameLast) {
-            html += '<h3 class="itemName infoText" style="margin: .25em 0 .5em;">' + name + '</h3>';
+            if (!layoutManager.mobile && parentNameHtml[1]) {
```

Comment:
why use specific index (`1` in this case)?..

Reply:
@JustAMan Explained above. I check that it's not undefined there because not all elements have two parents.